File: polivariable.c
#include <stdio.h>
int main(int argc, char *argv[]) {
	int A, B; 
	int A2, B2;
	int sumaA, sumaB;
		printf("= = PROGRAMA QUE SUMA DOS POLINOMIOS = =\n\n");
		printf("ESCRIBA COEFICIENTES DEL PRIMER POLINOMIO:\n");
		scanf("%d", &A);
		scanf("%d", &B);
		printf("\n\t EL RESULTADO ES: %dx + %d ",A,B);
		printf("\n\n ESCRIBA COEFICIENTES DEL SEGUNDO POLINOMIO:\n");
		scanf("%d", &A2);
		scanf("%d", &B2);
		printf("\n\t SU POLINOMIO ES: %dx + %d ",A2,B2);
		sumaA=A+A2;
		sumaB=B+B2;
		printf("\n\n LA SUMA DE LOS POLINOMIOS ES: %dx + %d",sumaA,sumaB);
	return 0;
}

File: polivariable.scala
object Polinomio {
    def main(args: Array[String]){
        println("Ingrese coeficientes del primer polinomio:")
		val a = readInt() 
		val b = readInt()
		println("Su polinomio es:")
        println(s"$a x + $b")
        println("\n\nIngrese coeficientes del segundo polinomio:")
		val a2 = readInt() 
		val b2 = readInt() 
		println("Su polinomio es:")
        println(s"$a2 x + $b2")
        val sumaa = a + a2
        val sumab = b + b2
		println(s"La suma de ambos polinomios es: $sumaa x + $sumab");
    }
}